src/main/kotlin/lotto/domain/LotteryMachine.kt

```diff
@@ -0,0 +1,31 @@
+package lotto.domain
+
+import java.math.RoundingMode
+
+object LotteryMachine {
+    private val LOTTERY_PRICE = Money(1_000)
+    private const val QUANTITY_SCALE = 0
+
+    fun issueLotteryTicket(money: Money): LotteryTicket {
+        val lotteryQuantity = getQuantity(money)
+        val lotteries = mutableListOf<Lottery>()
+        repeat(lotteryQuantity.toInt()) {
+            lotteries.add(generateNumbers())
+        }
+        return LotteryTicket(lotteries)
+    }
```

Comment:
repeat 도 좋지만 요런 것도 있지요 😉 
```suggestion
    fun issueLotteryTicket(money: Money): LotteryTicket {
        val lotteryQuantity = getQuantity(money)
        val lotteries = List(lotteryQuantity.toInt()) { generateNumbers() }
        return LotteryTicket(lotteries)
    }
```